packages/components/src/components/input/input.test.ts

```diff
@@ -81,6 +81,32 @@ describe('<sd-input>', () => {
     });
   });
 
+  describe('value min & max', () => {
+    it('should react on value within of range form date', async () => {
+      const el = await fixture<SdInput>(html`
+        <sd-input type="date" value="2023-06-10" max="2024-06-10" spellcheck></sd-input>
+      `);
+      expect(el.hasAttribute('data-valid')).to.be.true;
+    });
+
+    it('should react on value out of range form date', async () => {
```

Comment:
```suggestion
    it('should react on value out of range form date', async () => {
```

Please describe in the test, that this is about invalidity and the max value.